geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -450,6 +461,20 @@ pub trait GeyserPlugin: Any + Send + Sync + std::fmt::Debug {
         Ok(())
     }
 
+    /// Called when a ContactInfo is received from a node
+    #[allow(unused_variables)]
+    fn notify_node_update(&self, interface: &FfiContactInfoInterface) -> Result<()> {
+        Ok(())
+    }
+
+    /// Called when a node is removed from the network
+    /// TODO: may need to provide wrapper here? Also maybe ok
+    /// if we marshall to repr(c) for this...
+    #[allow(unused_variables)]
+    fn notify_node_removal(&self, pubkey: &FfiPubkey) -> Result<()> {
+        Ok(())
+    }
+
```

Comment:
also this is not a proper FFI interface since it is returning a Result<()>, so need to update...